src/parameterizations/vertical/MOM_kappa_shear.F90

```diff
@@ -794,7 +798,9 @@ subroutine kappa_shear_column(kappa, tke, dt, nzc, f2, surface_pres, hlay, dz_la
   do K=nzc,2,-1
     dist_from_bot = dist_from_bot + dz_lay(k)
     h_from_bot = h_from_bot + hlay(k)
-    I_L2_bdry(K) = ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
+    !I_L2_bdry(K) = ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
+    !               ((dist_from_top(K) * dist_from_bot) * (h_from_top(K) * h_from_bot))
+    I_L2_bdry(K) = 1/(CS%beta*CS%beta)*((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
```

Comment:
I can never remember the order of arithmetic precedence for multiplication and division in Fortran, and there is usually not reason to have multiple (expensive) divisions on the same line.  Please revise this to put the `CS%beta**2` term into the denominator, and write this line to have only a single division, either by moving `CS%beta**2` into the denominator or by storing the reciprocal of `CS%beta**2`.  Also, beta is not a very descriptive name; surely we can come up with something more descriptive than a Greek letter that is used elsewhere for something else for this variable name.

Reply:
thanks for these suggestions, I made the changes in another commit. I decided to save the inverse of the Lz rescale value. I renamed "beta" to "lz_rescale".

Reply:
When I started this improvement, I made a few changes to src/parameterizations/vertical/MOM_kappa_shear.F90:
```fortran
+ Line 722:  real :: beta     ! Is a new non-dimensional factor added by Vivek, Wei, Al  UW NOAA/PMEL
- Line 794:  dist_from_bot = 0.0 ; h_from_bot = 0.0 
+ Line 794:  dist_from_bot = 0.0 ; h_from_bot = 0.0 ; beta = 0.2
- Line 798:     I_L2_bdry(K) =  ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
                   ((dist_from_top(K) * dist_from_bot) * (h_from_top(K) * h_from_bot))
+ Line 798:     I_L2_bdry(K) =  1/(beta*beta) * ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
                   ((dist_from_top(K) * dist_from_bot) * (h_from_top(K) * h_from_bot))

```
Thank you very much for adding these changes.